modules/server/certbot/certbot.service.tmpl

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Certbot renewing ssl certificates for https
+After=docker.service,haveged.service,searchbroker.service,auth.service,mdr.service,nginx.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStartPre=/usr/bin/docker exec nginx mkdir -p /tmp/.well-known/acme-challenge
```

Comment:
don't like /tmp here. should be another directory. also don't like the idea to create a dir in another container

Reply:
I tried the last days many other ways but somehow it's not possible for certbot to create the .well-known/acme-challenge/ directory. Using "alias" instead of "root" in the nginx conf, also does not work.

Reply:
But why /tmp? /tmp is a special system directory. We shouldn't use it for our stuff. Especially because we mount a volume into /tmp.

Reply:
I could mount /var ? Or should we mount a directory inside default nginx/certbot install directory?

Reply:
Simply /certbot would be ok